api-report/container-runtime.api.md

```diff
@@ -474,7 +474,7 @@ export interface ISummarizer extends IEventProvider<ISummarizerEvents> {
     enqueueSummarize(options: IEnqueueSummarizeOptions): EnqueueSummarizeResult;
     readonly ISummarizer?: ISummarizer;
     // (undocumented)
-    run(onBehalfOf: string, disableHeuristics?: boolean): Promise<SummarizerStopReason>;
```

Comment:
Fun fact, `Summarizer` that implements `ISummarizer` does not even use `disableHeuristics` as a parameter in its `run` method, so I removed it here.